src/it/scala/com/wavesplatform/it/transactions/TransferTransactionSuite.scala

```diff
@@ -206,4 +208,22 @@ class TransferTransactionSuite extends BaseTransactionSuite with CancelAfterFail
 
     Await.result(f, waitCompletion)
   }
+
+  // probably should be moved elsewhere, just don't see any suite that fits the purpose right now
```

Comment:
Please don't leave TODOs/FIXMEs of any kind, either do the improvement if it's needed  or remove the letter